### PR TITLE
NIAD-1085: release fixes

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Please explain the changes you have made here.

## Jira Ticket

Please add a link to the related Jira ticket (if any).

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-02-22
```
Comment (n-r-b) [inline CHANGELOG.md:9]: Should this date be in the past or is this the date we cut the release branch?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Limited set of resources produced - Requester (FHIR Practitioner resource)
+
+### Changed
+n/a
+
+### Removed
+n/a
+
+### Known Issues
+- Information Disclosure [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415] in com.google.guava:guava@29.0-jre introduced by com.google.guava:guava@29.0-jre
```
Comment (n-r-b) [inline CHANGELOG.md:22]: I don't think we're using Guava in the adaptor or are these known issues for transitive dependencies?

**Review by saileshp-kainos — APPROVED** [review]
LGTM

**Review by antonlebedjko — APPROVED** [review]
Looks good

File: CHANGELOG.md (hunk 2 above)
Comment (bartek-sarul) [reply CHANGELOG.md:22]: it's indirect - that's one of the dependencies dependency

File: CHANGELOG.md (hunk 1 above)
Comment (bartek-sarul) [reply CHANGELOG.md:9]: branch cut off is fine

**Pull request merged**